src/Simulations/run.jl

```diff
@@ -102,11 +115,24 @@ end
 
 const ModelCallsite = Union{TendencyCallsite, UpdateStateCallsite}
 
+
+function time_step_or_skip!(sim)
+    model_callbacks = Tuple(cb for cb in values(sim.callbacks) if cb.callsite isa ModelCallsite)
+    Δt = aligned_time_step(sim, sim.Δt)
+    if Δt < sim.minimum_relative_step * sim.Δt
+        next_time = next_actuation_time(sim)
```

Comment:
FYI, `next_actuation_time(sim)` and `sim.model.clock.time + Δt` are equivalent. I chose to use the former here to avoid errors related to the addition operation. But let me know if I should change it to just use the latter.

Reply:
Doesn't it complicate the code to define `next_actuation_time` in addition to `aligned_time_step`? Things go wrong if you change one but not the other. It's usually best to have one "source" of reality / truth

Reply:
Fair enough. Changed it to the simpler `sim.model.clock.time + Δt`.